### ISSUE TITLE
MantraJS-atom vs Mantra-CLI

### ISSUE THREAD

**Issue by nadeemja** [mantrajs/mantrajs-atom-package#28]
Hi,

It appears that MantraJS-atom and Mantra-CLI are in disagreement about a few things.
1. Mantra-atom creates `index.js`, while mantra-cli creates `main.js` in server and client folders. 
   Using both Mantra-atom and Mantra-CLI therefore causes duplication errors for methods and publications in meteor server.
2. Mantra-atom requires ./configs/config in client/index.js (which it doesn't create and causes `unable to resolve` type errors), whereas Mantra CLI creates and requires ./configs/context
3. Mantra-atom adds file routes.js to core-module. Mantra CLI does not.

There might be other incongruencies.

Please advise on what we should do. :) 

PS. Thanks for two **awesome** tools. My life has improved <3

**Comment by macrozone** [comment]
Maybe the atom-package could use mantrajs-cli under the hood?

Can't say much about the 3 disagreements though. I think the mantra-sample application is the best guideline here

**Comment by tomitrescak** [comment]
If you read docs, the mantra atom package is fully configurable via yaml file, whatever you do not like can be changed with no hassle. You can change file names, file contents, directories, even add or reorganise panes. This is not possible with cli package. If you feel that these changes would benefit the whole community, just create a PR, like the recent one which changed stories folder to .stories folder.

**Comment by macrozone** [comment]
@tomitrescak , yeah the atom-package's yaml configs are great! Would be cool if the cli could use the same templates. @sungwoncho 's idea of having a .mantra-config-folder sounds also promising: https://talk.mantrajs.com/t/do-we-have-to-use-the-mantra-generated-eslintrc/178/3
